src/software/thunderscope/widget_setup_functions.py

```diff
@@ -141,6 +145,7 @@ def setup_gl_widget(
     gl_widget.add_layer(validation_layer)
     gl_widget.add_layer(trail_layer, False)
     gl_widget.add_layer(debug_shapes_layer, True)
+    gl_widget.add_layer(referee_layer, False)
```

Comment:
I think this layer should be on by default

Reply:
Done